test/vaults/integration/Deposit.t.sol

```diff
@@ -158,25 +158,6 @@ contract DepositTest is BaseTest {
         assertTrue(vault.maxDeposit(self) <= amount * 0.01e18);
     }
 
-    function testDepositWithPrepaymentFromGateway(uint256 amount) public {
```

Comment:
This test is hard to fix right now, and i don't believe it adds much: it really just tests the gateway subsidy logic instead of the deposit flow, which is already tested well within the Gateway unit tests.